packages/eslint-plugin/src/rules/consistent-type-imports.ts

```diff
@@ -93,12 +100,14 @@ export default util.createRule<Options, MessageIds>({
     {
       prefer: 'type-imports',
       disallowTypeAnnotations: true,
+      fixStyle: 'separate-type-imports',
     },
   ],
 
   create(context, [option]) {
     const prefer = option.prefer ?? 'type-imports';
     const disallowTypeAnnotations = option.disallowTypeAnnotations !== false;
+    const fixStyle = option.fixStyle ?? 'separate-type-imports';
```

Comment:
This makes sense for backwards compatibility, but it's unfortunate since the experience is so much better the other way. 😩